pkgs/by-name/wi/wifi-qr/package.nix

```diff
@@ -37,6 +41,12 @@ stdenvNoCC.mkDerivation (finalAttr: {
     qrencode
     xdg-utils
     zbar
+    # needed for cross
+    # TODO: somehow splice the packages in stdenvNoCC.initialPath and use that
```

Comment:
`__splicedPackages.stdenvNoCC.initialPath` doesn't work and `import ../../../stdenv/generic/common-path.nix { pkgs = __splicedPackages; }` got too ugly